be/src/exprs/aggregate_functions.cpp

```diff
@@ -25,6 +23,7 @@
 #include "runtime/string_value.h"
 #include "runtime/datetime_value.h"
 #include "exprs/anyval_util.h"
+#include "exprs/aggregate_functions.h"
```

Comment:
You'd better not move `#include "exprs/aggregate_functions.h"`, we put this first to make sure that all need includes is written in header files